app/src/main/java/com/hedvig/app/feature/claimdetail/ui/FakeWaveAudioPlayerCard.kt

```diff
@@ -63,80 +71,141 @@ fun FakeWaveAudioPlayerCard(
     audioPlayerState: AudioPlayerState,
     startPlaying: () -> Unit,
     pause: () -> Unit,
+    retryLoadingAudio: () -> Unit,
     waveInteraction: WaveInteraction,
+    modifier: Modifier = Modifier,
 ) {
-    Surface(
-        modifier = Modifier
-            .height(64.dp)
-            .fillMaxWidth(),
-        shape = MaterialTheme.shapes.medium,
-        color = if (isSystemInDarkTheme()) {
+    val color by animateColorAsState(
+        if (audioPlayerState is AudioPlayerState.Failed) {
+            MaterialTheme.colors.warning
+        } else if (isSystemInDarkTheme()) {
             MaterialTheme.colors.secondary
         } else {
             colorResource(R.color.lavender_200)
         },
-        contentColor = if (isSystemInDarkTheme()) {
+        animationSpec = tween(500)
+    )
+    val contentColor by animateColorAsState(
+        if (audioPlayerState is AudioPlayerState.Failed) {
```

Comment:
This can be moved to an extension on AudioPlayerState, or a separate function - would make this easier to read